README.md

```diff
@@ -18,28 +18,34 @@
     var express = require('express');
     var app = express();
 
-    // create the view engine with `react-engine`
-    var engine = require('react-engine').server.create({
-      reactRoutes: <string> /* pass in the path to react-router routes optionally */
-      performanceCollector: <function> /* optional function to collect perf stats */
-    });
+    app.use(function(req, res) {
```

Comment:
@sherodtaylor: the problem here is that you are trying to create a view engine for every request (which seems less acceptable if all you need is `res`). can you decouple the need for res?